src/Workspaces/CoreTest/WorkspaceTests/MSBuildWorkspaceTests.cs

```diff
@@ -2372,6 +2372,7 @@ public void TestProjectReferenceWithNoGuid()
             }
         }
 
+        [WorkItem(5668, "https://github.com/dotnet/roslyn/issues/5668")]
```

Comment:
I'll look. But then the coverage for that specific culture would be sparse anyway - the set of few tests that set the culture. shouldn't all tests be run on atleast one other culture than en-US on a regular basis?